wurst/systems/boards/ObserverBoard.wurst

```diff
@@ -179,14 +180,16 @@ function getObservers() returns LinkedList<player>
     return observers
 
 init
-    GameMode.onModeSelectionFinish() ->
+    registerGameStartEvent() ->
         let observers = getObservers()
         for observer in observers
-            let options = asList<ObserverBoardControl>(
-                new ObserverBoardVisionControl(observer),
-                new ObserverBoardZoomControl(observer)
-            )
-            new ObserverBoard(observer, options)
+            nullTimer() () ->
+                ScoreBoard.board.display(observer, true)
+            // let options = asList<ObserverBoardControl>(
+            //     new ObserverBoardVisionControl(observer),
+            //     new ObserverBoardZoomControl(observer)
+            // )
+            // new ObserverBoard(observer, options)
```

Comment:
Would be nice to integrate this into the new board or something instead of just commenting that out...